docker/prognostic_run/Dockerfile

```diff
@@ -26,19 +26,19 @@ RUN make -C /tmp/fortran-build/FV3 wrapper_build &&\
     python3 -c 'import fv3gfs.wrapper'
 
 COPY external/vcm /fv3net/external/vcm
-RUN pip3 install --no-dependencies /fv3net/external/vcm
+RUN pip3 install --no-dependencies -e /fv3net/external/vcm
```

Comment:
I also modified PYTHONPATH below. Did this for redundancy, since if a user tries to modify the PYTHONPATH it's likely to break things in hard-to-detect ways.

Reply:
Yes, the `-e` will not work unless you have a `egg-info` folder created in the directory which happens if you `pip install -e` that directory, in which case PYTHONPATH will matter. But if a user does `PYTHONPATH=/my/custom/code` it would break all the packages if not for the redundant things-are-develop-installed. So this way there would need to be two failures for the installs to break.